### PR TITLE
DTSPB-4295 Applicant Name Enhancement

### PR DESCRIPTION
**Pull request by jamiealbertelli**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] update CSS version to force refresh if any changes are introduced

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by IswaryaPepakayala — DISMISSED** [review]

File: app/steps/ui/applicant/aliasreason/template.html [hunk 1]
```diff
@@ -59,6 +73,7 @@
             }
         ]
     }) }}
+    {% endcall %}
```
Comment (jamiealbertelli) [inline app/steps/ui/applicant/aliasreason/template.html:76]: Just double checking this line should be here and not above L33.

File: app/steps/ui/applicant/nameasonwill/template.html [hunk 2]
```diff
@@ -4,6 +4,20 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% block form_content %}
+    <h1 class="govuk-heading-l">{{ content.reminder | safe }}</h1>
```
Comment (jamiealbertelli) [inline app/steps/ui/applicant/nameasonwill/template.html:7]: Made this heading large and line 29 medium to match UCD's design. Leaving this comment as a reminder to check that the font sizes are acceptable.

File: test/component/applicant/testAliasReason.js [hunk 3]
```diff
@@ -26,10 +26,12 @@ describe('applicant-alias-reason', () => {
                 }
             };
 
+            const contentToExclude = ['reasonForNameChangeQuestionSummary', 'optionDifferentSpelling'];
```
Comment (jamiealbertelli) [inline test/component/applicant/testAliasReason.js:29]: Added 'optionDifferentSpelling' to content to exclude. Happy to look into if incorrect.

File: charts/probate-frontend/values.yaml [hunk 4]
```diff
@@ -21,7 +21,7 @@ nodejs:
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     PAYMENT_API_URL: http://payment-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     VALIDATION_SERVICE_URL: http://probate-business-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/validate
-    ORCHESTRATOR_SERVICE_URL : http://probate-orchestrator-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    ORCHESTRATOR_SERVICE_URL : http://probate-orchestrator-service-pr-988-java
```
Comment (IswaryaPepakayala) [inline charts/probate-frontend/values.yaml:24]: Don't forget to change this before release

**Pull request merged**